app/assets/stylesheets/_mixins.scss

```diff
@@ -11,21 +11,6 @@
   #{$property}: var(--#{$cssVariable}, #{$fallback});
 }
 
-/**
-*
-* Makes working with themeable CSS variables easier. Used when
-* a CSS variable requires !important as a postfix of the CSS property's
-* value where the CSS variable is being used.
-*
-* $property: the CSS property to theme.
-* $cssVariable: The CSS variable that will store the themed value.
-* $fallback: The value to use as a fallback but as well for the CSS variable default value.
-*/
-@mixin themeable-important($property, $cssVariable, $fallback) {
-  #{$property}: #{$fallback} !important;
-  #{$property}: var(--#{$cssVariable}, #{$fallback}) !important;
-}
-
```

Comment:
It's been used in only one place but apparently it's no longer needed so dropping it.

Reply:
Phew! Sweet.